releasenotes/notes/python-310-support-f7040d4ff5e4022e.yaml

```
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Added support for Python 3.10.
```

A bit more verbiage here would probably be useful. Maybe something like:

```suggestion
    Added support for running with Python 3.10. Starting with this release Qiskit Dynamics now supports
    running with Python 3.7 through 3.10.
```